Task10_Auth/src/main/java/Mapper.java

```diff
@@ -0,0 +1,20 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import reseponsesDTO.TokenDto;
+
+public class Mapper {
+    private TokenDto tokenDto = new TokenDto();
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+
+    public TokenDto tokenStringToObject(String response ) {
```

Comment:
Correct code will be like:
`	
	public static <T> T entityToObj(HttpEntity entity, Class<T> clazz) {
		String body;
		try {
			body = EntityUtils.toString(entity, "UTF-8");
		} catch (IOException e) {
			throw new RuntimeException("Can't transform http entity to string");
		}
		try {
			return objectMapper.readValue(body, clazz);
		} catch (JsonProcessingException e) {
			throw new RuntimeException("Can't map to the object of " + clazz.getName());
		}
	}
`
And usage:
`TokenDto tokenDto = Mapper.entityToObj(response.getEntity(), TokenDto.class);`

Reply:
done